core/src/it/scala/org/ensime/core/RefactoringHandlerSpec.scala

```diff
@@ -44,11 +47,11 @@ class RefactoringHandlerSpec extends EnsimeSpec
     val analyzer = analyzerRef.underlyingActor
 
     val procId = 1
-    val result = analyzer.handleRefactorRequest(
+    val result = Await.result(analyzer.handleRefactorRequest(
```

Comment:
if you rebase, it's been added to `EnsimeSpec` already (you will probably get a few trivial conflicts, best to read that PR to see what changed)